sqlglot/helper.py

```diff
@@ -62,6 +62,15 @@ def apply_index_offset(expressions, offset):
     return expressions
 
 
+def parent(expression, kind):
```

Comment:
This is not currently used anywhere right? do you need it? If so perhaps rename to find_ancestor. Also minor but I would change the local variable below from expression to ancestor and return None when not found